components/org.wso2.carbon.identity.oauth2.grant.rest.endpoint/src/main/java/org/wso2/carbon/identity/oauth2/grant/rest/endpoint/util/ConfigUtil.java

```diff
@@ -0,0 +1,151 @@
+package org.wso2.carbon.identity.oauth2.grant.rest.endpoint.util;
```

Comment:
There are multiple files with no license header. Can you check all and fix them?

Reply:
fixed in 733ee6bbfe71c91b9d1c6ad17be4cd4feb38fcbc